packages/utils/src/psi-runner.js

```diff
@@ -19,20 +19,22 @@ class PsiRunner {
 
   /**
    * @param {string} url
-   * @param {{psiApiKey?: string, psiApiEndpoint?: string, psiStrategy?: 'mobile'|'desktop'}} [options]
+   * @param {{psiApiKey?: string, psiApiEndpoint?: string, psiStrategy?: 'mobile'|'desktop', categories?: Array<'performance' | 'accessibility' | 'best-practices' | 'pwa' | 'seo'>}} [options]
```

Comment:
these options are all redundantly prefixed with `psi` because the CLI `collect` command can also use them, I think it might be worth doing at this level for `psiCategories` as well :) (but in `psi-client`, definitely leave without prefix)

WDYT?

Reply:
Yeah, sounds very reasonable to stay consistent.